Fluent/Post_Porcess.ipynb

```diff
@@ -8,53 +8,97 @@
     "\n",
-    "In order to validate the obtained results, the temporal variation of Lift and Drag coefficients is studied. The Drag coefficient ($C_D$) and Lift Coefficient ($C_L$) are defined as:\n",
+    "As the uniform streamwise flow interacts with the wall of rotating cylinder it will result into a drag force (in the streamwise direction of the flow) and a lift force (in perpendicular direction of the flow) on the body of rotating cylinder. Normalizing the magnitude of these forces and investigate their temporal evolution is another post-processing step to see whether or not the obtained numerical solutions are valid and has converged. Cylinder's drag coefficient ($C_D$) and lift Coefficient ($C_L$) are defined as:\n",
     "\n",
     "$$ C_D = \\frac{F_D}{0.5*\\rho*V^2*A_{proj}}, $$\n",
     "\n",
     "$$ C_L = \\frac{F_L}{0.5*\\rho*V^2*A_{proj}}, $$\n",
     "\n",
-    "where $F_D$ and $F_L$ are Drag and Lift forces respectively. These forces are due to pressure and viscous forces acting on the cylinder. The quantity $0.5*\\rho*V^2$ is the available momentum in the free stream multiplied by the projected area. Hence, these coefficients implies the ratio of corresponding lift and drag forces to available force in the free stream flow field.\n",
-    "User can set up Drag and Lift coefficient plots versus iterations before starting the simulation using the path `Solution/Monitors/Create/Lift/Drag`, select the cylinder surface as **zone**. When the simulation starts, $C_D$ and $C_L$ are plotted against number of iterations. These plots are shown below:\n",
+    "where $F_D$ and $F_L$ are drag and lift forces respectively. These are net forces due to pressure and viscous forces acting on the body of the cylinder. The quantity $0.5*\\rho*V^2$ is the available momentum in the free stream multiplied by the projected area, which makes it the available force \"acting\" on the cylinder by the free stream. Hence, these coefficients implies the ratio of corresponding net lift and drag net forces to available force in the free stream flow field. User can set up Drag and Lift coefficient history plots versus number of iterations before starting the simulation using the path <span style=style=\"background-color:lightgrey;\">Solution/Monitors/Create/Lift/Drag</span>, select the cylinder surface as **zone**. When the simulation starts, $C_D$ and $C_L$ are plotted against number of iterations. These plots are shown below:\n",
     "\n",
     "<img src=\"./Images/LiftDragTimeStep.png\\\" width=\"800\">\n",
```

Comment:
You have a high resolution of these plots. Use them. If exporting from PPT is reducing the resolution use the figures separately here.